### PR TITLE
[docs] updated governance doc

### PR DESCRIPTION
**Pull request by 0xchloe**
### Description
Added more details to the governance doc. Lmk if I'm missing anything, would be happy to add more : )

### CODE REVIEW

**Comment by clay-aptos** [comment]
And when this lands, I will update See what's new to summarize it:
https://aptos.dev/whats-new-in-docs

Thanks, @0xchloe !

File: developer-docs-site/docs/concepts/governance.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ sources={{
 
 ## Who can propose
 
-- To either propose or vote, you must stake but you are not required to run a validator node. However, we recommend that you be a validator with a stake and be a part of the validator set. 
+- To either propose or vote, you must stake, but you are not required to run a validator node. However, we recommend that you be a validator with a stake and be a part of the validator set.
```
Comment (movekevin) [inline developer-docs-site/docs/concepts/governance.md:36]: nit: Rationale for why becoming a validator is recommended is that users will also gain rewards.

File: developer-docs-site/docs/concepts/governance.md [hunk 2]
```diff
@@ -45,3 +45,24 @@ sources={{
 :::tip
 Each stake pool can only be used to vote on each proposal exactly once.
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements. To resolve a proposal, you can use the `aptos governance execute-proposal` command from Aptos CLI. 
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain.
```
Comment (movekevin) [inline developer-docs-site/docs/concepts/governance.md:54]: I would omit "the Aptos Labs team"

File: developer-docs-site/docs/concepts/governance.md [hunk 3]
```diff
@@ -45,3 +45,24 @@ sources={{
...
 Each stake pool can only be used to vote on each proposal exactly once.
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements. To resolve a proposal, you can use the `aptos governance execute-proposal` command from Aptos CLI. 
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain. 
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues).
```
Comment (movekevin) [inline developer-docs-site/docs/concepts/governance.md:55]: Can we add a callout that they should also follow the template there?

File: developer-docs-site/docs/concepts/governance.md [hunk 4]
```diff
@@ -45,3 +45,24 @@ sources={{
...
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues). 
+To keep up with new AIPs, check the #aip-announcements channel on Aptos' discord channel. 
+To view and vote on on-chain proposals, go to [`Aptos' Governance website`](https://governance.aptosfoundation.org/). 
+
+## Technical Implementation of Aptos Governance
+The majority of governance logic are in [`aptos_governance.move and voting.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources). 
+The `aptos_governance` module outlines how users can interact with Aptos Governance. It's the external-facing module of the Aptos on-chain governance process and contains logic and checks that are specific to Aptos Governance.
+The `voting` module is the Aptos governance standard that can be used by DAOs on the Aptos chain to create their own on-chain governance process.
+
+If you are thinking about creating a DAO on Aptos, you can refer to `aptos_governance`'s usage of the `voting` module as an example.
```
Comment (movekevin) [inline developer-docs-site/docs/concepts/governance.md:64]: Can you add more details on the functionalities of the voting module and may be just simple explanation of how a DAO can leverage them? We'll add a more detailed tutorial soon.

File: developer-docs-site/docs/concepts/governance.md (hunk 1 above)
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:36]:
```suggestion
- To either propose or vote, you must stake, but you are not required to run a validator node. However, we recommend that you run validator with a stake as part of the validator set to gain rewards from your stake.
```

File: developer-docs-site/docs/concepts/governance.md (hunk 2 above)
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:54]:
```suggestion
AIPs are proposals created by the Aptos community to improve the operations and development of the Aptos blockchain. 
```

File: developer-docs-site/docs/concepts/governance.md [hunk 5]
```diff
@@ -45,3 +45,24 @@ sources={{
 :::tip
 Each stake pool can only be used to vote on each proposal exactly once.
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements. To resolve a proposal, you can use the `aptos governance execute-proposal` command from Aptos CLI.
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:50]:
```suggestion
- Anyone can resolve an on-chain proposal that has passed voting requirements by using the `aptos governance execute-proposal` command from Aptos CLI. 
```

File: developer-docs-site/docs/concepts/governance.md (hunk 3 above)
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:55]:
```suggestion
To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues) using the [template](https://github.com/aptos-foundation/AIPs/blob/main/TEMPLATE.md)
```

File: developer-docs-site/docs/concepts/governance.md [hunk 6]
```diff
@@ -45,3 +45,24 @@ sources={{
...
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements. To resolve a proposal, you can use the `aptos governance execute-proposal` command from Aptos CLI. 
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain. 
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues). 
+To keep up with new AIPs, check the #aip-announcements channel on Aptos' discord channel.
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:56]:
```suggestion
To keep up with new AIPs, check the #aip-announcements channel on [Aptos' discord channel](https://discord.gg/aptoslabs). 
```

File: developer-docs-site/docs/concepts/governance.md [hunk 7]
```diff
@@ -45,3 +45,24 @@ sources={{
...
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain. 
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues). 
+To keep up with new AIPs, check the #aip-announcements channel on Aptos' discord channel. 
+To view and vote on on-chain proposals, go to [`Aptos' Governance website`](https://governance.aptosfoundation.org/). 
+
+## Technical Implementation of Aptos Governance
+The majority of governance logic are in [`aptos_governance.move and voting.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources).
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:60]:
```suggestion
The majority of the governance logic is in [`aptos_governance.move and voting.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources). 
```

**Review by davidiw — APPROVED** [review]

File: developer-docs-site/docs/concepts/governance.md [hunk 8]
```diff
@@ -45,3 +45,24 @@ sources={{
...
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements by using the `aptos governance execute-proposal` command from Aptos CLI. 
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain. 
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues) using the [template](https://github.com/aptos-foundation/AIPs/blob/main/TEMPLATE.md)
+To keep up with new AIPs, check the #aip-announcements channel on [Aptos' discord channel](https://discord.gg/aptoslabs).
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/governance.md:56]:
```suggestion
To keep up with new AIPs, check the `#aip-announcements` channel on [Aptos' discord channel](https://discord.gg/aptoslabs). 
```

**Review by davidiw — APPROVED** [review]

**Comment by davidiw** [comment]
> And when this lands, I will update See what's new to summarize it: https://aptos.dev/whats-new-in-docs
> 
> Thanks, @0xchloe !

@clay-aptos it landed

**Comment by clay-aptos** [comment]
> > And when this lands, I will update See what's new to summarize it: https://aptos.dev/whats-new-in-docs
> > Thanks, @0xchloe !
> 
> @clay-aptos it landed

Thanks! Announcement PR here:
https://github.com/aptos-labs/aptos-core/pull/5862

Also see a lingering question on an accompanying code comment I have above. Thanks again.

**Pull request merged**